vmicore/src/vmi/LibvmiInterface.cpp

```diff
@@ -51,8 +51,6 @@ void LibvmiInterface::initializeVmi(const std::function<void()>& postInitializat
         throw VmiInitError(initError);
     }
 
-    kernelJson = vmi_get_kernel_json(vmiInstance);
```

Comment:
That's not at all what I meant though. :D I was specifically talking about `wstringConverter`. That's why I've annotated the corresponding line and not this one.